### PR TITLE
Feature/darpa

### PR DESCRIPTION
**Pull request by alexmillane**
@clanegge Can we merge this? Or is this still under construction?

### CODE REVIEW

File: mav_utils_msgs/package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>mav_utils_msgs</name>
```
Comment (rikba) [inline mav_utils_msgs/package.xml]: I'd rather call this `mav_system_msgs`

File: mav_utils_msgs/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+project(mav_utils_msgs)
+
+find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
+include_directories(include ${catkin_INCLUDE_DIRS})
+
+
+add_message_files(
+ FILES
+ CpuInfo.msg
+ ProcessInfo.msg
```
Comment (rikba) [inline mav_utils_msgs/CMakeLists.txt]: You don't need to explicitly name these. add_message_files will automatically detect all messages in [/msg folder](https://github.com/ros/genmsg/blob/f9dad2f897fc9bafec038ed8a74986ba766e9e9d/cmake/genmsg-extras.cmake.em#L111). Makes it easier to maintain.

File: mav_state_machine_msgs/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mav_state_machine_msgs)
+
+find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
+include_directories(include ${catkin_INCLUDE_DIRS})
+
+
+add_message_files(
+  FILES
+  StartStopTask.msg
```
Comment (rikba) [inline mav_state_machine_msgs/CMakeLists.txt]: You don't need to explicitly name these. add_message_files will automatically detect all messages in [/msg folder](https://github.com/ros/genmsg/blob/f9dad2f897fc9bafec038ed8a74986ba766e9e9d/cmake/genmsg-extras.cmake.em#L111). Makes it easier to maintain.

File: mav_state_machine_msgs/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+)
+add_service_files(
+  FILES
+  RunTaskService.srv
```
Comment (rikba) [inline mav_state_machine_msgs/CMakeLists.txt]: Same for srv files.

File: mav_utils_msgs/CMakeLists.txt (hunk 2 above)
Comment (clanegge) [reply mav_utils_msgs/CMakeLists.txt]: This is magic! Did not know that

File: mav_utils_msgs/package.xml (hunk 1 above)
Comment (clanegge) [reply mav_utils_msgs/package.xml]: Changed

**Pull request merged**